addon-SV_OptimizedListQueries.xml

```diff
@@ -42,24 +49,24 @@
     </option>
     <option option_id="sv_convquery_threshold" edit_format="spinbox" data_type="integer" can_backup="1">
       <default_value>150</default_value>
-      <edit_format_params>min=0</edit_format_params>
+      <edit_format_params></edit_format_params>
       <sub_options></sub_options>
       <relation group_id="performance" display_order="10110"/>
     </option>
     <option option_id="sv_forumquery_threshold" edit_format="spinbox" data_type="integer" can_backup="1">
       <default_value>150</default_value>
-      <edit_format_params>min=0</edit_format_params>
+      <edit_format_params></edit_format_params>
       <sub_options></sub_options>
       <relation group_id="performance" display_order="10100"/>
     </option>
     <option option_id="sv_likequery_threshold" edit_format="spinbox" data_type="integer" can_backup="1">
       <default_value>20</default_value>
-      <edit_format_params>min=0</edit_format_params>
+      <edit_format_params></edit_format_params>
       <sub_options></sub_options>
       <relation group_id="performance" display_order="10130"/>
     </option>
     <option option_id="sv_membersonline_query" edit_format="onoff" data_type="boolean" can_backup="1">
-      <default_value>0</default_value>
+      <default_value>1</default_value>
```

Comment:
I've set this default to 0 because this feature introduces breaking changes to XenForo functionality in a obvious user visible way